core/app/models/spree/customer_return.rb

```diff
@@ -1,5 +1,6 @@
 module Spree
   class CustomerReturn < Spree::Base
+    include Spree::Core::NumberGenerator.new(prefix: 'CR', length: 9)
```

Comment:
So to override this you would just call this include again in your decorator rather than overriding #generate_number ?